src/status_im/contexts/wallet/common/transaction_settings/priority_fee/view.cljs

```diff
@@ -25,10 +25,10 @@
 
 (defn view
   []
-  (let [priority-fee (rf/sub [:wallet.send/tx-settings-priority-fee])
-        max-base-fee (rf/sub [:wallet.send/tx-settings-max-base-fee])
-        spectrum     {:low  (rf/sub [:wallet.send/tx-settings-suggested-min-priority-fee])
-                      :high (rf/sub [:wallet.send/tx-settings-suggested-max-priority-fee])}
+  (let [priority-fee (rf/sub [:wallet/tx-settings-max-base-fee])
```

Comment:
Again base fee used instead of priority fee. @mohsen-ghafouri, is it an accident or they are changed intentionally?

Reply:
Thanks @vkjr  for spotting this, i might mess up during changes, will fix it